src/test/scala/com/lunatech/iamin/rest/VersionRouteSpec.scala

```diff
@@ -0,0 +1,36 @@
+package com.lunatech.iamin.rest
+
+import cats.effect.IO
+import org.http4s.implicits._
+import com.lunatech.iamin.rest.version.VersionResource
+import com.lunatech.iamin.rest.version.impl.VersionHandler
+import org.http4s._
+import org.specs2.matcher.MatchResult
+
+object BuildInfo {
+  def version: String = "test-version"
+}
+
+class VersionRouteSpec extends org.specs2.mutable.Specification {
+
+  "Version route" >> {
+    "return 200" >> {
+      uriReturns200()
+    }
+    "return the current app's version" >> {
+      uriReturnsVersion()
+    }
+  }
+
+  private[this] val retVersion: Response[IO] = {
```

Comment:
If we want to prefix responses, let's go with `res` instead of `ret`, `ret` implies a return value while this is clearly a response value.